releasenotes/notes/reliable-dual-shipping-by-default-8e6641654ffbf177.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Make logs agent ``additional_endpoints`` reliable by default.
+    This can be disabled by setting setting ``is_reliable: false` 
```

Comment:
```suggestion
    This can be disabled by setting ``is_reliable: false``
```

Reply:
Could you confirm if `is_reliable: false` has the correct syntax for back-ticks? It looks like one may be missing

Reply:
Good catch!  It was not correct before.